recipes/ai-rag-amx-ubuntu/terraform.tfstate

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
We need to remove this file from the commit and add *.tfstate to the .gitignore file